packages/cashier/src/Containers/__tests__/deposit.spec.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Deposit from '../deposit';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+jest.mock('Components/Error/virtual', () => () => <div>Virtual</div>);
+jest.mock('Components/Error/cashier-locked', () => () => <div>CashierLocked</div>);
+jest.mock('Components/Error/funds-protection', () => () => <div>FundsProtection</div>);
+jest.mock('Components/Error/deposit-locked', () => () => <div>DepositsLocked</div>);
+jest.mock('Components/Form/crypto-transactions-history', () => () => <div>CryptoTransactionsHistory</div>);
+jest.mock('Components/Error/error', () => () => <div>Error</div>);
+jest.mock('../crypto-deposit', () => () => <div>CryptoDeposit</div>);
+jest.mock('Components/cashier-container', () => () => <div>CashierContainer</div>);
+jest.mock('Components/CashierDefault/cashier-default', () => () => <div>CashierDefault</div>);
+
+describe('<Deposit />', () => {
+    const props = {
+        currency: 'USD',
+        current_currency_type: 'fiat',
+        error: { is_ask_uk_funds_protection: false, message: '' },
+        is_cashier_locked: false,
+        is_crypto_transactions_visible: false,
+        is_deposit: false,
+        is_deposit_locked: false,
+        is_eu: false,
+        is_loading: false,
+        is_switching: false,
+        is_system_maintenance: false,
+        is_virtual: false,
+        onMount: jest.fn(),
+        recentTransactionOnMount: jest.fn(),
+        setActiveTab: jest.fn(),
+        setIsDeposit: jest.fn(),
+        setErrorMessage: jest.fn(),
+        setSideNotes: jest.fn(),
+    };
+
+    it('should render <Loading /> component', () => {
+        const { rerender } = render(<Deposit {...props} is_switching is_loading={false} iframe_url='' />);
+
+        expect(screen.getByText('Loading')).toBeInTheDocument();
+
+        rerender(<Deposit {...props} is_switching={false} is_loading iframe_url='' />);
```

Comment:
what is expected here since you are calling `rerender`?

Reply:
@farrah-deriv omg, i missed expect under rerender, thx for the notice